src/Exception/UnprocessableEntityHttpException.php

```diff
@@ -11,4 +11,15 @@
  */
 class UnprocessableEntityHttpException extends ClientErrorHttpException
 {
+    /**
+     * Returns the errors of the response if there are any
+     *
+     * @return array
+     */
+    public function getResponseErrors()
+    {
+        $decodedBody = json_decode($this->getResponseBody(), true);
```

Comment:
It's indirectly checked in the line below with isset($decodedBody['errors']). If the response is not a json, an empty array will be returned.